includes/Integrations/Core_Themes_Support.php

```diff
@@ -77,9 +105,29 @@ public function extend_theme_support() {
 	 */
 	public function embed_web_stories() {
 		$customizer = new Customizer();
+		$this->enqueue_style( $this->style_handle );
 		echo $customizer->render_stories(); // phpcs:ignore -- WordPress.Security.EscapeOutput.OutputNotEscaped - Escaped web stories HTML.
```

Comment:
I don't like all the `is-showing-header-stories` classes in the CSS. Let's have a wrapper div here instead. Something like this, perhaps with a better class name:

```suggestion
		$customizer = new Customizer();
		$this->enqueue_style( $this->style_handle );
		?>
		<div class="web-stories-theme-header-section">
		<?php echo $customizer->render_stories(); // phpcs:ignore -- WordPress.Security.EscapeOutput.OutputNotEscaped - Escaped web stories HTML. ?>
		</div>
		<?php
```

Then we can target this in CSS with something like `.web-stories-theme-header-section .web-stories-list { ... }` and not `.is-showing-header-stories > .web-stories-list`

Reply:
I didn't want to add an extra layer but adding this surely removes the need for the extra class on `body`. I'll update.